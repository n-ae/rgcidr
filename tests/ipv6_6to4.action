64:ff9b::/96 tests/ipv4_mapped_ipv6.given
