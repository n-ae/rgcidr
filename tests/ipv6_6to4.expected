64:ff9b::192.168.1.1
